src/main/java/porcelli/me/git/integration/githook/push/GitHook.java

```diff
@@ -1,13 +1,14 @@
 package porcelli.me.git.integration.githook.push;
 
+import static java.util.Comparator.comparing;
```

Comment:
please remove unused code.

Reply:
Comparator.comparing is used in line 106, removing it will cause compilation errors..

Reply:
same here: you formatted the code. please revert the changes of this file and it should be ok

Reply:
Done